backend/src/nodes/utils/pil_utils.py

```diff
@@ -47,6 +49,26 @@ def resize(
     return np.array(pimg).astype("float32") / 255
 
 
+def rotate(
+    img: np.ndarray, angle: float, interpolation: int, expand: int, fill: int
+) -> np.ndarray:
+    """Perform PIL rotate"""
+
+    # Select how to fill negative space that results from rotation
+    c = get_h_w_c(img)[2]
+    if fill == 0:
+        fill_color = (0,) * c
+    elif fill == 1:
+        fill_color = (0,) * c if c < 4 else (0, 0, 0, 255)
+    else:
+        img = convert_to_BGRA(img, c)
+        fill_color = (0, 0, 0, 0)
+
+    pimg = Image.fromarray((img * 255).astype("uint8"))
+    pimg = pimg.rotate(angle, interpolation, expand, fillcolor=fill_color)  # type: ignore
```

Comment:
`interpolation` is incorrect-ish. You pass in a value from `InterpolationMethod` which are not guaranteed to be PIL interpolation methods. The only reason this works is that the values defined in `InterpolationMethod` happen to line up with PIL interpolation methods.

Please use the `INTERPOLATION_METHODS_MAP` to get a PIL interpolation method.

Since we no longer use OpenCV for resizing, we could also remove the InterpolationMethod class, but that should be its own PR.